src/Microsoft.FeatureManagement.AspNetCore/TagHelpers/FeatureTagHelper.cs

```diff
@@ -38,12 +39,14 @@ public class FeatureTagHelper : TagHelper
         public string Variant { get; set; }
 
         /// <summary>
-        /// Creates a feature tag helper.
+        /// Creates a feature tag helper. Takes both a feature manager and a variant feature manager for backwards compatibility.
```

Comment:
I don't think that needs to be in the class summary. Might be helpful as a code comment.